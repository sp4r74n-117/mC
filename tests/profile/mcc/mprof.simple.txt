function main
 called from ??
 took avg: 7260444 stddev: 0 cycles
function sum
 called from main
 took avg: 72533 stddev: 5976 cycles
