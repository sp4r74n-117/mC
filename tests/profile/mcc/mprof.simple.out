0x8048a18 0x8048986 76612
0x8048a18 0x8048986 70144
0x8048a18 0x8048986 72580
0x8048a18 0x8048986 70096
0x8048a18 0x8048986 78552
0x8048a18 0x8048986 70132
0x8048a18 0x8048986 95680
0x8048a18 0x8048986 70100
0x8048a18 0x8048986 70116
0x8048a18 0x8048986 70076
0x8048a18 0x8048986 70076
0x8048a18 0x8048986 78624
0x8048a18 0x8048986 74164
0x8048a18 0x8048986 70112
0x8048a18 0x8048986 70104
0x8048a18 0x8048986 70136
0x8048a18 0x8048986 70108
0x8048a18 0x8048986 99736
0x8048a18 0x8048986 75336
0x8048a18 0x8048986 75392
0x8048a18 0x8048986 73020
0x8048a18 0x8048986 75768
0x8048a18 0x8048986 71240
0x8048a18 0x8048986 71696
0x8048a18 0x8048986 75636
0x8048a18 0x8048986 70072
0x8048a18 0x8048986 72460
0x8048a18 0x8048986 82240
0x8048a18 0x8048986 70116
0x8048a18 0x8048986 70072
0x8048a18 0x8048986 91000
0x8048a18 0x8048986 70120
0x8048a18 0x8048986 70080
0x8048a18 0x8048986 70080
0x8048a18 0x8048986 70080
0x8048a18 0x8048986 70100
0x8048a18 0x8048986 70076
0x8048a18 0x8048986 70076
0x8048a18 0x8048986 70076
0x8048a18 0x8048986 70084
0x8048a18 0x8048986 70076
0x8048a18 0x8048986 70076
0x8048a18 0x8048986 70072
0x8048a18 0x8048986 70084
0x8048a18 0x8048986 71384
0x8048a18 0x8048986 70076
0x8048a18 0x8048986 70072
0x8048a18 0x8048986 70092
0x8048a18 0x8048986 70088
0x8048a18 0x8048986 70072
0x8048a18 0x8048986 76784
0x8048a18 0x8048986 70088
0x8048a18 0x8048986 70072
0x8048a18 0x8048986 70076
0x8048a18 0x8048986 87708
0x8048a18 0x8048986 80072
0x8048a18 0x8048986 70068
0x8048a18 0x8048986 70072
0x8048a18 0x8048986 70080
0x8048a18 0x8048986 70088
0x8048a18 0x8048986 70076
0x8048a18 0x8048986 70076
0x8048a18 0x8048986 70080
0x8048a18 0x8048986 70088
0x8048a18 0x8048986 70080
0x8048a18 0x8048986 70064
0x8048a18 0x8048986 70076
0x8048a18 0x8048986 70092
0x8048a18 0x8048986 70076
0x8048a18 0x8048986 70088
0x8048a18 0x8048986 70080
0x8048a18 0x8048986 70096
0x8048a18 0x8048986 70072
0x8048a18 0x8048986 70084
0x8048a18 0x8048986 99592
0x8048a18 0x8048986 70088
0x8048a18 0x8048986 70104
0x8048a18 0x8048986 70080
0x8048a18 0x8048986 70080
0x8048a18 0x8048986 88296
0x8048a18 0x8048986 70076
0x8048a18 0x8048986 70076
0x8048a18 0x8048986 70076
0x8048a18 0x8048986 70072
0x8048a18 0x8048986 70080
0x8048a18 0x8048986 70080
0x8048a18 0x8048986 72424
0x8048a18 0x8048986 70080
0x8048a18 0x8048986 70088
0x8048a18 0x8048986 73692
0x8048a18 0x8048986 70088
0x8048a18 0x8048986 73692
0x8048a18 0x8048986 70088
0x8048a18 0x8048986 73692
0x8048a18 0x8048986 70076
0x8048a18 0x8048986 70076
0x8048a18 0x8048986 70084
0x8048a18 0x8048986 70076
0x8048a18 0x8048986 70080
0x8048a18 0x8048986 70076
0x804894b 0xf75bd72e 7260444
